exercises/exercise-2.md

```diff
@@ -1,5 +1,45 @@
 ---
 layout: guide
+banner: images/ex2-banner.jpg
 ---
 
 # Exercise 2
+
+In exercise 2, we are going to make a program that is capable of receiving chat messages over bluetooth, and printing those messages to a terminal on your laptop using the USB connection.
+
+1. **Create a new project on the [online compiler](https://developer.mbed.org/compiler) based on the [Location Puck](todo) example.**
+Remember to set a unique ID in the puck `init` function, as this will make your life much easier later on.
+You can find a guide on how to create new projects in the guides section if you get stuck.
```

Comment:
There is a pending prq on "how to use import libraries" that also says how one to create a new project. I guess we can either rename that guide or change the wording here?